bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/URLAudioStream.java

```diff
@@ -90,7 +90,10 @@ private InputStream createInputStream() throws AudioException {
                 os.write(req.getBytes());
                 is = shoutCastSocket.getInputStream();
             } else {
-                is = streamUrl.openStream();
+                // getInputStream() method is more error-proof than openStream(),
+                // because openStream() does openConnection().getInputStream(), 
+                // which opens a new connection and does not reuse the old one.
+                is = connection.getInputStream();
```

Comment:
Made a PR with this change: https://github.com/eclipse/smarthome/pull/3119. Ignored the tests, related to removing the change from this branch.